app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    persons = []
+
+    for one_person in people:
+        persons.append(Person(one_person["name"], one_person["age"]))
```

Comment:
Use list comprehensions. Also, one_person is bad naming, because person is already one